staging/src/k8s.io/client-go/tools/remotecommand/resize.go

```diff
@@ -16,18 +16,10 @@ limitations under the License.
 
 package remotecommand
 
-// TerminalSize and TerminalSizeQueue was a part of k8s.io/kubernetes/pkg/util/term
-// and were moved in order to decouple client from other term dependencies
+import apimachinery "k8s.io/apimachinery/pkg/util/remotecommand"
 
-// TerminalSize represents the width and height of a terminal.
-type TerminalSize struct {
-	Width  uint16
-	Height uint16
-}
+// TODO: move all imports of these types to use the new package.
```

Comment:
I think this code can return to client-go once streamtranslator moves to client-go

Reply:
Done